install.php

```diff
@@ -1,3 +1,40 @@
 <?php
-$_GET["mod"] = "install";
-include_once("index.php");
+
+//raise error reporting - could be interesting and this is autmomatically reset when index.php is executed
```

Comment:
I would favor comments like `// Raise error ...`.
The reason is readability. It is like a sentence in an article or in a book. There you expect also a "Whitespace" in front of the sentence (like `This is super. And here it goes ...`. And not `This is super.And here it goes ...`. What do you think about this?

Furthermore, `autmomatically` => `automatically`

Reply:
Oh. This applies to all `//` comments in this PR.